grpc/proto/grpc_api.proto

```diff
@@ -77,6 +78,13 @@ message AgentHello {
 message Goodbye {
 }
 
+/**
+* A message providing information about the workloads to be added after the agent connects.
```

Comment:
Let's be a bit more precise here and describe that this is a response to the agent hello. Afterwards we can say what it provides.

Reply:
Done.